devel-tools/cirrus-status/lambda_function.py

```diff
@@ -0,0 +1,83 @@
+import calendar
+import logging
+import os
+from datetime import datetime, timezone
+
+import requests
+from gql import Client, gql
+from gql.transport.aiohttp import AIOHTTPTransport
+
+# Configure logging
+logger = logging.getLogger()
+logger.setLevel("INFO")
+
+BALANCE_LIMIT = 2500.0
+
+# Configuration from environment variables
+GRAPHQL_ENDPOINT = os.getenv("GRAPHQL_ENDPOINT")
+GRAPHQL_TOKEN = os.getenv("GRAPHQL_TOKEN")
+SLACK_WEBHOOK_URL = os.getenv("SLACK_WEBHOOK_URL")
```

Comment:
We should probably `assert` that we got a value for all of these, otherwise something down the line might fail in cryptic ways.